components/inspectit-ocelot-configurationserver-ui/src/components/views/configuration/scope-wizard/TypeMatcher.js

```diff
@@ -68,7 +68,7 @@ const TypeMatcher = ({ typeMatcher, onTypeMatcherChange, onShowClassBrowser }) =
         <div className="row-center">
           <Dropdown style={{ width: '7rem' }} value={typeMatcher.type} options={types} onChange={(e) => setState('type', e.value)} />
           <label className="inner-label" htmlFor="which">
-            which name
+            name
```

Comment:
I personally preferred simply removing the "which", since it still expresses what it is supposed to just fine and then it is just a bit less somewhat redundant text in the UI. I'd also agree with replacing it with a "whose" which would be analogous to the way it is written in the overview after finishing creating a scope with the scope wizard :)